services/logging-operator/3.17.9/defaults/cm.yaml

```diff
@@ -98,10 +98,10 @@ data:
         secret.reloader.stakater.com/reload: logging-operator-logging-fluentd-tls
     fluentbit:
       image:
-        # Explicitly use the default image. This should be updated when logging-operator is upgraded.
+        # Explicitly specify the version here. This should be updated when logging-operator is upgraded.
```

Comment:
should we bump the version of logging-operator now just in case we don't end up bumping the chart next release?

Reply:
logging operator was bumped in https://github.com/mesosphere/kommander-applications/pull/863 
i bumped the pinned image version in values overrides in this PR.